tjmonopix2/analysis/plotting.py

```diff
@@ -262,7 +272,7 @@ def create_scurves_plot(self, scan_parameter_name='Scan parameter'):
                 scan_parameter_range = self.scan_parameter_range
             else:
                 scan_parameter_name = '$\\Delta$ VCAL'
-                electron_axis = False  # TODO: True after calibration code is done
+                electron_axis = self.electron_axis
```

Comment:
Since we determine `electron_axis` before we can get rid of the declarations here and just pass `self.electron_axis` to `_plot_scurves()`

Reply:
Not entirely True. There is an additional if-else statement checking for specific scans and setting this flag accordingly. In TJ-Monopix2, this is absolutely not used but just copy and paste legacy from BDAQ53.
If I adapt this, I will get rid of the if clauses as well.